lms/envs/production.py

```diff
@@ -1153,7 +1153,7 @@
     )
 CALIPER_DELIVERY_ENDPOINT = ENV_TOKENS.get('CALIPER_DELIVERY_ENDPOINT')
 CALIPER_DELIVERY_AUTH_TOKEN = AUTH_TOKENS.get('CALIPER_DELIVERY_AUTH_TOKEN')
-
+CALIPER_KAFKA_SETTINGS = ENV_TOKENS.get('CALIPER_KAFKA_SETTINGS')
```

Comment:
will `CALIPER_KAFKA_SETTINGS` be a dict? do we need to define this in other settings file ?

Reply:
it will be configured in `lms.env.json` and Yes it is a dict.
```
 "CALIPER_KAFKA_SETTINGS": {
        "END_POINT": "<ip-address>:<port-number>",
        "TOPIC_NAME": "dummy name",
        "ERROR_REPORT_EMAIL": "dummy@gmail.com",
        "NUMBER_OF_RETRIES": 3
    }
```

Reply:
done.
Replaced with `MAXIMUM_RETRIES`.